examples/step-67/step-67.cc

```diff
@@ -73,12 +74,14 @@ namespace Euler_DG
   // the final time up to which we run the simulation, and a variable
   // `output_tick` that specifies in which intervals we want to write output
   // (assuming that the tick is larger than the time step size).
-  constexpr unsigned int testcase             = 0;
+  constexpr unsigned int testcase             = 1;
   constexpr unsigned int dimension            = 2;
   constexpr unsigned int n_global_refinements = 3;
   constexpr unsigned int fe_degree            = 5;
   constexpr unsigned int n_q_points_1d        = fe_degree + 2;
 
+  constexpr bool use_ecl = true;
```

Comment:
Please document this above, maybe also in combination with a more expressive variable name.